model/patch_lifecycle.go

```diff
@@ -256,7 +256,7 @@ func MakePatchedConfig(ctx context.Context, env evergreen.Environment, p *patch.
 			SetErrorSender(level.Error, grip.GetSender()).SetOutputSender(level.Info, grip.GetSender()).
 			Directory(workingDirectory).Run(ctx)
 		if err != nil {
-			return nil, errors.Wrap(err, "could not run patch command")
```

Comment:
We shouldn't say "on evergreen.yml" because this path is configurable by project; we could just generically say "Evergreen configuration file" or something. It's also maybe possible that this isn't where the error comes from so it could be good to say "possibly" or something like that.

Reply:
done